### PR TITLE
yacreader: init at 9.5.0, libunarr: init at 1.0.1

### PR DESCRIPTION
**Pull request by cocreature**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/graphics/yacreader/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    url = "https://github.com/YACReader/yacreader/releases/download/${version}/${name}-src.tar.xz";
+    sha256 = "0cv5y76kjvsqsv4fp99j8np5pm4m76868i1nn40q6hy573dmxwm6";
+  };
+
+  nativeBuildInputs = [ qmake pkgconfig ];
+  buildInputs = [ poppler libunarr libGLU qtmultimedia qtscript ];
+  propagatedBuildInputs = [ qtquickcontrols qtgraphicaleffects qtdeclarative ];
+
+  meta = {
+    description = "A comic reader for cross-platform reading and managing your digital comic collection.";
```
Comment (c0bw3b) [inline pkgs/applications/graphics/yacreader/default.nix]: No period at the end of the description
https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

File: pkgs/development/libraries/libunarr/default.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  src = fetchurl {
+    url = "https://github.com/selmf/unarr/releases/download/v${version}/unarr-${version}.tar.xz";
+    sha256 = "1db500k6w90qn6qb4j3zcczailmmv81q9lv4bwq516hbncg5p4sl";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/selmf/unarr;
+    description = "A lightweight decompression library with support for rar, tar and zip archives.";
```
Comment (c0bw3b) [inline pkgs/development/libraries/libunarr/default.nix]: Same thing

**Review by c0bw3b — CHANGES_REQUESTED** [review]
Please add `enableParallelBuilding = true;` to both packages

It builds fine on nixos-18.09 otherwise

**Comment by cocreature** [comment]
Thanks for the feedback @c0bw3b. I think I addressed all of your requests.

**Comment by c0bw3b** [comment]
@GrahamcOfBorg build libunarr yacreader

**Review by c0bw3b — APPROVED** [review]

**Comment by c0bw3b** [comment]
Yheah much faster builds!

**Pull request merged**